libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/DistanceFormatter.java

```diff
@@ -48,9 +51,12 @@
    * @param context  from which to get localized strings from
    * @param language for which language
    * @param unitType to use, or NONE_SPECIFIED to use default for locale country
+   * @param roundingIncrement increment by which to round small distances
    */
   public DistanceFormatter(Context context, @NonNull String language,
-                           @NonNull @DirectionsCriteria.VoiceUnitCriteria String unitType) {
+                           @NonNull @DirectionsCriteria.VoiceUnitCriteria String unitType,
+                           @NavigationConstants.RoundingIncrement int roundingIncrement) {
```

Comment:
Can we also update `DistanceFormatter#shouldUpdate` to consider `roundingIncrement` as well?